src/main/java/seedu/duke/entities/Dish.java

```diff
@@ -29,7 +29,7 @@ private static String toTitleCase(String givenString) {
     }
 
     private static boolean isValidName(String name) {
-        return !(name == null || name.equals(""));
+        return !(name == null || name.length() == 0);
```

Comment:
you do know theres a `isEmpty` method or smth right lol

Reply:
also im not sure how name would ever be null if you use getString method of InputParser

Reply:
so technically, you can just check for name.isEmpty() in the constructor and throw IllegalArgumentException there, dunnid a extra method to check valid name.

Reply:
Okay I go change

Reply:
Will handle it in another PR, as we will have to change the JUnit tests also

Reply:
ok ill leave it to u

Reply:
> so technically, you can just check for name.isEmpty() in the constructor and throw IllegalArgumentException there, dunnid a extra method to check valid name.

still need the boolean there cuz I used `isValidName` for some assertion, maybe only change to `name.isEmpty()`?

Reply:
> > so technically, you can just check for name.isEmpty() in the constructor and throw IllegalArgumentException there, dunnid a extra method to check valid name.
> 
> still need the boolean there cuz I used `isValidName` for some assertion, maybe only change to `name.isEmpty()`?

I think we can still use name.isEmpty() for assertions, but then the JUnit tests I encountered some problems with our tests so might have to change that part

But assertions shld be fine by using `assert !(name.isEmpty())`

Reply:
> > > so technically, you can just check for name.isEmpty() in the constructor and throw IllegalArgumentException there, dunnid a extra method to check valid name.
> > 
> > 
> > still need the boolean there cuz I used `isValidName` for some assertion, maybe only change to `name.isEmpty()`?
> 
> I think we can still use name.isEmpty() for assertions, but then the JUnit tests I encountered some problems with our tests so might have to change that part
> 
> But assertions shld be fine by using `assert !(name.isEmpty())`

I guess that's I tested null value haha (i ll delete that later), then it's fine.